src/Umbraco.Web.UI.Client/src/assets/lang/en.ts

```diff
@@ -2368,7 +2368,7 @@ export default {
 		checkFailed: 'Check failed',
 		openBackofficeSearch: 'Open backoffice search',
 		openCloseBackofficeHelp: 'Open/Close backoffice help',
-		openCloseBackofficeProfileOptions: 'Open/Close your profile options',
```

Comment:
FYI, removing a localization key is considered a breaking-change, as 3rd-party extensions may be using them. 🤷 (So, we leave them in until we can do a larger clean-up in a major release, v17/v18).